mssqlcli/mssqlcliclient.py

```diff
@@ -24,7 +24,13 @@ def __init__(self, sql_tools_client, server_name, user_name, password,
                  authentication_type=u'SqlLogin', database=u'master', owner_uri=None, multiple_active_result_sets=True,
                  encrypt=None, trust_server_certificate=None, connection_timeout=None, application_intent=None,
                  multi_subnet_failover=None, packet_size=None, **kwargs):
+
         self.server_name = server_name
+        if ',' in server_name:
+            self.host, self.port = self.server_name.split(',')
```

Comment:
since host and port are for prompt display purposes, can you rename them to prompt_host and prompt_port? This way, it's clear we're not using the values when connecting to the target database.